promote/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Convox Promote
+description: Promote a release of a previously built Convox app
```

Comment:
everywhere "previously built" is used is a bit confusing.. does that mean in this workflow? in another system?  These actions should have super simple descriptions, e.g. "Create a release", "Promote a release", "Run a command"